pkgs/development/compilers/adoptopenjdk-icedtea-web/default.nix

```
@@ -1,14 +1,14 @@
 { stdenv, fetchFromGitHub, cargo, rustc, autoreconfHook, jdk, glib, xulrunner, zip, pkgconfig, npapi_sdk, bash, bc }:
 
 stdenv.mkDerivation rec {
-  name = "icedtea-web-${version}";
+  name = "adoptopenjdk-icedtea-web";
```

This is missing the `p`.
```suggestion
  pname = "adoptopenjdk-icedtea-web";
```

---

I'm not familiar with this pattern. I've always seen and used `name`, do we need `pname` and `version` then?

---

see https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv.xml#L34

---

Thanks for the pointer!